cmd/gateway/main.go

```diff
@@ -238,6 +240,7 @@ func startGateway() {
 	traceEverything := viper.GetBool("trace-everything")
 	debug := viper.GetBool("debug")
 	cpuprofile := viper.GetString("cpuprofile")
+	authFromAws := viper.GetBool("cb-auth-from-awssm")
```

Comment:
Note that we need to support fetching secrets from all 3 major cloud providers (aws, azure and gcp).